### PR TITLE
Add a doc: Install Litmus on KubeSphere

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>

**What this PR does / why we need it**:
Hi there, I am Serena from the KubeSphere Team. I add a document about how to install Litmus on KubeSphere. Please take a look, thanks!

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for DCO to be passed
-   [x] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/versioned_docs/version-1.10.0/kubesphere-litmus.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
+---
+id: KubeSphere-litmus
+title: Installtion of LitmusChaos on KubeSphere
```
Comment (ksatchit) [inline website/versioned_docs/version-1.10.0/kubesphere-litmus.md]:
```suggestion
title: Installation of LitmusChaos on KubeSphere
```

Comment (serenashe) [reply website/versioned_docs/version-1.10.0/kubesphere-litmus.md]: Changed. Thanks!

**Comment by serenashe** [comment]
I updated the doc according to the comment. Please take a look, thank :)

**Review by ksatchit — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Comment by ispeakc0de** [comment]
@serenashe Can we add this doc to the sidebars? Also, we can add it to the master version?

**Comment by serenashe** [comment]
> @serenashe Can we add this doc to the sidebars? Also, we can add it to the master version?

Sure! But should I create a new pull request for this or should you guys just cherry-pick this one?

**Comment by neelanjan00** [comment]
@serenashe will you just add the `kubesphere-litmus.md` file to the `master` directory as well? That would suffice. 🙂

**Review by neelanjan00 — APPROVED** [review]

**Pull request merged**